src/components/search-input/search-input.module.scss

```diff
@@ -0,0 +1,46 @@
+.form {
+    position: relative;
+    left: 1em;
+    width: 16em;
+
+    @media (max-width: 768px) {
+        width: 10em;
+    }
+}
+
+.input {
+    width: 100%;
+    padding: 0 1.5em;
+    border: none;
+    background: none;
+    font-size: 0.9em;
+
+    &::placeholder {
+        font-size: inherit;
+    }
```

Comment:
Does this have any effect?

Reply:
apparently not. missed it while was fiddling to match Dana's expectations.
deleted this rule